indexer-app/src/cmr/indexer/data/concepts/keyword_util.clj

```diff
@@ -16,12 +16,28 @@
   "Defines Regex to split strings with special characters into multiple words for keyword searches."
   #"[!@#$%^&()\-=_+{}\[\]|;'.,\\\"/:<>?`~* ]")
 
+(defn- wrapped-keyword?
+  "Checks if the field-value about to be processed is a term surrounded by parens or brackets.
+   If it is, we want to add the usual keywords, but also add the unwrapped field-value on its own."
+  [field-value]
+  (let [first-char (first field-value)]
```

Comment:
This is a syntax error.

Reply:
That's embarassing. I'll blame it on it being a Friday afternoon